annotell-input-api/examples/lidars_and_cameras_seq_with_pre_annotations.py

```diff
@@ -84,20 +85,11 @@ def run(
     # Simulate waiting for scene to be created since this is a async process
     # The rest needs to be run once the scene creation has finished
     print(f"Sleeping 10 seconds in order to wait for the scene to be created")
```

Comment:
Durations are out of sync 

```suggestion
    print(f"Sleeping 20 seconds in order to wait for the scene to be created")
```